tests/validation/tests/v3_api/scripts/pipeline/Jenkinsfile_ontag_matrix

```diff
@@ -109,7 +114,7 @@ if ( true == via_webhook() && (!(rancher_version ==~ rancher_version_regex)) ) {
                 sh "./tests/v3_api/scripts/build.sh"
               }
 
-              try {
+              try {              
```

Comment:
looks like you have some whitespace here?

Reply:
Nice catch! I'll fix that.